cmd/qat_plugin/README.md

```diff
@@ -188,16 +188,16 @@ For more information, refer to:
 #### Deploy the pod
 
 In the pod specification file, add container resource request and limit.
-For example, `qat.intel.com/generic: <number of devices>` for a container requesting QAT devices.
+For example, `qat.intel.com/cy: <number of devices>` for a container requesting QAT devices.
 
 For a DPDK-based workload, you may need to add hugepage request and limit.
 
 ```bash
-$ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/qat_dpdk_app/base/
+$ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/qat_dpdk_app/crypto-perf/
```

Comment:
If we change this, it essentially becomes the same as "Automated test run". Can you work on this DPDK demo section a bit more (additional context: I've wanted to get rid of that "manual test run" completely since it's outdated and not working)

Reply:
Sure, let me update readme after carefully reading a bit more.

Reply:
this part looks ok